### PR TITLE
Force pull of images on `make start-environment`

### PR DESCRIPTION
**Pull request by jsoriano**
When using snapshots, image could be outdated, force pulling images before starting environment.

Ported from #8204

### CODE REVIEW

File: libbeat/scripts/Makefile [hunk 1]
```diff
@@ -377,6 +377,7 @@ build-image: write-environment
 # To use it for running the test, set ES_HOST and REDIS_HOST environment variable to the ip of your docker-machine.
 .PHONY: start-environment
 start-environment: stop-environment
+	${DOCKER_COMPOSE} pull --include-deps
```
Comment (ruflin) [inline libbeat/scripts/Makefile]: I assume this will use the cached layers if they are available and identical?

Comment (jsoriano) [reply libbeat/scripts/Makefile]: Yes, it still reuses local layers.

Comment (jsoriano) [reply libbeat/scripts/Makefile]: But I see build times increased some minutes... maybe this is normal if we were re-using images too much.

Comment (ruflin) [reply libbeat/scripts/Makefile]: I think its ok as it gets us on the same page with travis. also if intermediary images used, the increase should only be once a day

**Pull request merged**